specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/examples/quantumWorkspacesDelete.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "workspaceName": "quantumworkspace1",
+    "resourceGroupName": "quantumResourcegroup",
+    "subscriptionId": "1C4B2828-7D49-494F-933D-061373BE28C2",
+    "api-version": "2019-11-04-preview"
+  },
+  "responses": {
+    "200": {
+      "description": "Successful."
+    },
+    "202": {
```

Comment:
>202 [](start = 5, length = 3)

This is not a correct response for an async delete. Please see [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#delete-resource-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#delete-resource-asynchronously) for details. (There should be no body on 202 and headers should be returned that tell the client where to pol for deletion status.)

Reply:
Swagger was wrong here, API is behaving as expected. Updated.